docs/source/llms/tracing/index.rst

```diff
@@ -706,6 +725,16 @@ and the :py:meth:`mlflow.client.MlflowClient.delete_trace_tag` method to remove
     client.delete_trace_tag(request_id="12345678", key="tag_key")
 
 
+Async Logging
+-------------
+
+By default, MLflow Tracing are logged synchronously. This may introduce a performance overhead when logging Traces, especially when your MLflow Tracking Server is running on a remote server. If the performance overhead is a concern for you, you can enable **asynchronous logging** for tracing in MLflow 2.16.0 and later.
```

Comment:
```suggestion
By default, MLflow Traces are logged synchronously. This may introduce a performance overhead when logging Traces, especially when your MLflow Tracking Server is running on a remote server. If the performance overhead is a concern for you, you can enable **asynchronous logging** for tracing in MLflow 2.16.0 and later.
```